src/utils/text.tsx

```diff
@@ -19,3 +19,7 @@ export const decamelizeText = (string: string, noUcFirst?: true, separator?: str
     return ucFirstText(decamelize(string, { separator: separator ? separator : ' ' }));
   return decamelize(string, { separator: separator });
 };
+
+export const widthText = (string: string, xWidth: number) => {
```

Comment:
Can we document what these kind of utilities do? Based on the name (I'd suggest "textWidth") the width of the text is being calculated?

Reply:
✅ => 📝 ADD function docs